ct/hev-socks5-server.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: miviro
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://github.com/heiher/hev-socks5-server
+
+# App Default Values
+APP="hev-socks5-server"
+# Name of the app (e.g. Google, Adventurelog, Apache-Guacamole"
+TAGS="proxy,socks5"
+# Tags for Proxmox VE, maximum 2 pcs., no spaces allowed, separated by a semicolon ; (e.g. database | adblock;dhcp)
+var_cpu="2"
+# Number of cores (1-X) (e.g. 4) - default are 2
+var_ram="1024"
+# Amount of used RAM in MB (e.g. 2048 or 4096)
+var_disk="4"
+# Amount of used disk space in GB (e.g. 4 or 10)
+var_os="debian"
+# Default OS (e.g. debian, ubuntu, alpine)
+var_version="12"
+# Default OS version (e.g. 12 for debian, 24.04 for ubuntu, 3.20 for alpine)
+var_unprivileged="1"
+# 1 = unprivileged container, 0 = privileged container
```

Comment:
Can you remove all the comments here?
```suggestion
APP="hev-socks5-server"
TAGS="proxy,socks5"
var_cpu="2"
var_ram="1024"
var_disk="4"
var_os="debian"
var_version="12"
var_unprivileged="1"
```